.github/actions/run-integration-tests/action.yml

```diff
@@ -82,12 +88,9 @@ runs:
         echo "IntegrationTestsJob: Start"
         set -o pipefail
         make \
-          KUBERNETES_DISTRIBUTION=${{ inputs.kubernetes_distribution }} \
-          KUBERNETES_ARCHITECTURE=${{ inputs.kubernetes_architecture }} \
-          DNSTESTER_IMAGE=${{ inputs.dnstester_image }} \
-          GADGET_REPOSITORY=${{ inputs.gadget_repository }} \
-          GADGET_TAG=${{ inputs.gadget_tag }} \
-          -o kubectl-gadget integration-tests |& tee integration.log & wait $!
+          IG_PATH=/home/runner/work/inspektor-gadget/inspektor-gadget/kubectl-gadget \
+          IG_RUNTIME=kubernetes \
+          -o kubectl-gadget integration-tests integration-test-gadgets |& tee integration.log & wait $!
```

Comment:
Could we split the steps for builtin gadgets and image based gadgets? IMO it looks clearer to have them side by side instead of combined in a single step and (way too long) output

Reply:
I also think it's cleaner to have independent steps, but it seems much more complex. The logic to catch the signals and perform the cleanup will have to be duplicated as well as the logic to publish the test results. IMHO we can live with the tests on the same step for now.

Reply:
I think everyone is waiting to rip out the buitin gadget and all legacy stuff :D

Reply:
Yes, we are! I think it'll help us a lot! For the moment, I'll stick to this solution, acknowledging that it's not the better but a good tradeoff.